t-654 Functions  medium.py

```diff
@@ -0,0 +1,9 @@
+# Напишіть функцію для визначення, чи рік високосний чи ні.
+def vysokosnyj_year(year:int):
+    if year%4 == 0 and year%100 != 0:
+        print(f'{year} є високосним роком ')
+    elif year%400 == 0:
```

Comment:
you do not need this elif. it is same as % 4

Also, the formating of code is not according to Pep8. Do not forget about ctrl+alt+L